tests/conftest.py

```diff
@@ -140,40 +151,36 @@ def cluster_teardown_finalizer():
     with open(install_config, "w") as f:
         f.write(install_config_str)
 
-    # Download installer
-    installer = get_openshift_installer(
-        config.DEPLOYMENT['installer_version']
-    )
     # Download client
     get_openshift_client()
 
     # Deploy cluster
-    log.info("Deploying cluster")
-    run_cmd(
-        f"{installer} create cluster "
-        f"--dir {cluster_path} "
-        f"--log-level {log_cli_level}"
-    )
+    deployer.deploy(log_cli_level)
+    deployer.add_volume()
+
+
+@pytest.fixture(scope="session", autouse=True)
+def ocs_cluster(cluster):
```

Comment:
I thought that this will be handled by deployment class in which we will have separation form OCP and OCS deployment.

Reply:
Done, Now a single fixture 'cluster' will take care of both OCS and OCP.